content/docs/start/experiments-trail/experiments.md

```diff
@@ -0,0 +1,319 @@
+---
+title: 'Get Started: Experiments'
+---
+
+# Get Started with Experiments
+
+<abbr>Experiments</abbr> proliferate quickly in ML projects where there are many
+parameters to tune or other permutations of the code. We can organize such
+projects and keep only what we ultimately need with `dvc experiments`. DVC can
+track experiments for you so there's no need to commit each one to Git. This way
+your repo doesn't become polluted with all of them. You can discard experiments
+once they're no longer needed.
+
+Previously, we learned how to tune [ML pipelines](/doc/start/data-pipelines) and
```

Comment:
I think one of the main goals of using separate "Trails" (I love the idea and this name!) - is to make them completely independent. I would not distract people by these back-references. We can simplify this.

Reply:
These two paragraphs are mostly from the older docs. I'll rewrite them. Thank you.

Reply:
Removed the references in 4f56d8f4